libkodimote/songs.cpp

```diff
@@ -210,10 +210,7 @@ void Songs::playItem(int row)
     } else {
         pItem.setAlbumId(m_albumId);
     }
-    Kodi::instance()->audioPlayer()->playlist()->clear();
-    Kodi::instance()->audioPlayer()->playlist()->addItems(pItem);
-//    Kodi::instance()->audioPlayer()->playlist()->playItem(row);
-    Kodi::instance()->audioPlayer()->playItem(row);
+    Kodi::instance()->audioPlayer()->open(pItem, resume);
```

Comment:
Good catch (actually, I don't have albums in my lib, so couldn't have found it).

I wanted to have a "unified API", that's why I've added the "open" call to both, so it doesn't matter if you have an audio or video player, you could always call "open" on it.